packages/citation-js-utils/src/index.ts

```diff
@@ -281,6 +283,7 @@ export function getCitationRenderers(data: CSL[]): CitationRenderer {
       });
       // Trim the DOIs and URLs (these are encoded) on load
       if (c.URL) c.URL = c.URL.replace(/^(%20)*/, '').replace(/(%20)*$/, '');
+      if (!c.issued && c.accessed) c.issued = c.accessed;
```

Comment:
If we remove this line, then the _inline_ citation still has a year, but the reference still has "n.d." with "Accessed..." - looks like:

![image](https://github.com/user-attachments/assets/c49d5c01-65fe-4b07-ad13-f829075ac419)

Possibly this is better...? Not sure.

Reply:
I think this is the one we want. I have asked @mepa from SciPy to chime in!

Reply:
This is better, and honest, and much like what you'd see on Wikipedia, which deals with lots of url refs.